### DISCUSSION TITLE
Surface reconstruction

### DISCUSSION THREAD

**Discussion by Hassan-Bahrami** [InteractiveComputerGraphics/SPlisHSPlasH#163] [Q&A]
Hi,

I'm looking for a efficient way of surface reconstruction SPH method. I cannot find related topics in the docs. Does [SPlisHSPlasH] have built-in surface reconstruction method?
Suppose I have a set of particles (SPH) and I'm going to recosntruct surface in Python dynamically, is it possible to do it by SPlisHSPlasH?

Thanks in advance!

**Comment by janbender** [comment]
SPlisHSPlasH has not surface reconstruction yet. But you could take a look at the project of one of my PhD students: 

https://github.com/w1th0utnam3/splashsurf
